tutorials/plot_background_statistics.py

```diff
@@ -0,0 +1,566 @@
+#   |                   |          Null hypothesis            |
+#   |                   +------------------+------------------+
+#   |                   |       True       |       False      |
+#   +==========+========+==================+==================+
+#   |          |        | Type I error     | Correct          |
+#   |          | Yes    |   False positive |   True positive  |
+#   + Reject   +--------+------------------+------------------+
+#   |          |        | Correct          | Type II error    |
+#   |          | No     |   True Negative  |   False negative |
+#   +----------+--------+------------------+------------------+
+#
+# To combat this problem, multiple methods exist. Typically these
+# provide control over either the:
+#
+# 1. `Familywise error rate (FWER) <fwer>`_
+#      The probability of making one or more type I error:
+#      :math:`p(N_{\mathrm{Type\ I}} >= 1)`.
+# 2. `False discovery rate (FDR) <fdr>`_
+#      The expected proportion of rejected null hypotheses that are false:
+#      :math:`N_{\mathrm{Type\ I}} / N_{\mathrm{reject}}`.
+#
+# We cover some techniques that control FWER and FDR below.
+#
+# Bonferroni correction
+# ^^^^^^^^^^^^^^^^^^^^^
+# Perhaps the simplest way to deal with multiple comparisons, `Bonferroni
+# correction<https://en.wikipedia.org/wiki/Bonferroni_correction>`_
+# conservatively multiplies the p-values by the number of comparisons to
+# control the FWER.
+
+titles.append('Bonferroni')
+ts.append(ts[-1])
+ps.append(bonferroni_correction(ps[0])[1])
+mccs.append(True)
+plot_t_p(ts[-1], ps[-1], titles[-1], mccs[-1])
+
+###############################################################################
+# False discovery rate (FDR) correction
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# Typically FDR is performed with the Benjamini/Hochberg procedure, which
+# is less restrictive than Bonferroni correction for large numbers of
+# comparisons (fewer type II errors) but provides less strict control of
+# errors (more type I errors).
+
+titles.append('FDR')
+ts.append(ts[-1])
+ps.append(fdr_correction(ps[0])[1])
+mccs.append(True)
+plot_t_p(ts[-1], ps[-1], titles[-1], mccs[-1])
+
+###############################################################################
+# Non-parametric resampling test with a maximum statistic
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# **Non-parametric resampling tests** can also be used to correct for multiple
+# comparisons. In its simplest form, we again do permutations using
+# exchangeability under the null hypothesis, but this time we take the
+# *maximum statistic across all tests* in each permutation to form the
+# null distribution. The p-value for each vertex from the veridical data
+# is then given by the proportion of null distrubtion values
+# that were smaller.
+#
+# This method has two important features:
+#
+# 1. It controls FWER.
+# 2. It is non-parametric. Even though our initial test statistic
```

Comment:
Hm, ok ... I'd still say it might be useful to state why one would want to do this at the beginning.